### ISSUE TITLE
Kernel always dies when using shaprpy

### ISSUE THREAD

**Issue by WardClaerhout** [NorskRegnesentral/shapr#410]
When trying to run the example code in a local Jupyter Notebook (Python) my kernel always dies. I have already tried running the shapr package in Rstudio and that works fine. Anyone else encountering the same problems or knows how to deal with them?

**Comment by martinju** [comment]
Yes, shaprpy is currently not working properly. We are working on a huge code refractoring. Shaprpy should be back working with https://github.com/NorskRegnesentral/shapr/pull/402 within a few days. Sorry about the issue.

**Comment by martinju** [comment]
This should be working well for several months now.
